### ISSUE TITLE
No .gitignore

### ISSUE THREAD

**Issue by Lashuk1729** [crespoter/djangotutorial_buggyversion#7]
Dear Contributors,

Always remember to add `.gitignore` in your repo. Many unwanted files (as its clearly visible in this commit 92849b50e89f80078c0a64f45985242ad6ad9aea ) will be added. Please don't hurt my coding feels this way.
Add `.gitignore` FFS.

Regards,
lashuk1729

**Comment by SriHarshaGajavalli** [comment]
@Lashuk1729 

I would agree with your point of having .gitignore. Please do add it and send it as a PR referencing this issue.

**Comment by crespoter** [comment]
Assigning @Lashuk1729
